src/NHSD.BuyingCatalogue.Ordering.Domain/Common/ValueObject.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace NHSD.BuyingCatalogue.Ordering.Domain.Common
+{
+    public abstract class ValueObject

```

Comment:
should probably have some unit tests

Reply:
Taken the unit tests for Buying Catalogue